static/nodejs_server/map_tile.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Functions for getting tile result for a map tile
+ */
+
+// This import is unused in this file, but needed for draw functions
+import * as Canvas from "canvas";
```

Comment:
why only scatter and map need Canvas?

Reply:
map and scatter use the canvas for creating the color scale legend. The other tiles don't need this